e2e/config.yaml.sample

```diff
@@ -1,9 +1,11 @@
 # Configuration for RamenDR End to End testing.
 
 ---
-# Git repository URL containing application manifests to be deployed on
-# the clusters.
-giturl: "https://github.com/RamenDR/ocm-ramen-samples.git"
+# Git repository url and branch containing application manifests
+# to be deployed on the clusters.
+repo:
+  url: "https://github.com/RamenDR/ocm-ramen-samples.git"
```

Comment:
Generally we don't need quoting for strings, but I think the ":" may confuse the yaml parser, so we should keep this quoted. We ned to fix this in ramenctl init.